spec/spec_helper.rb

```diff
@@ -1,46 +1,68 @@
 require 'rubygems'
 require 'bundler/setup'
 require 'rspec'
-require 'fake_dynamo'
-require 'aws-sdk'
 
-Aws.config.update({
+require 'English'
+
+require 'aws-sdk-dynamodb'
+require 'dynamodb-mutex'
+
+Aws.config.update(
   credentials: Aws::Credentials.new(
     'your_access_key_id',
     'your_secret_access_key'
   ),
   endpoint: 'http://localhost:4567'
-})
+)
 
-require 'dynamodb-mutex'
+AMAZON_LOCAL_DYNAMODB_URL =
+  'https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/dynamodb_local_latest.tar.gz'.freeze
+DYNAMODB_JAR_DIR = 'resources'.freeze
+DYNAMODB_TMP_PATH = '/tmp/dynamodb_local_latest.tar.gz'.freeze
+
+def fetch_amazon_dynamodb_local
+  Dir.mkdir(DYNAMODB_JAR_DIR) unless Dir.exist?(DYNAMODB_JAR_DIR)
+
+  return if File.exist?("./#{DYNAMODB_JAR_DIR}/DynamoDBLocal.jar")
+
+  raise "Error downloading #{AMAZON_LOCAL_DYNAMODB_URL}" unless \
+    system("wget #{AMAZON_LOCAL_DYNAMODB_URL} -O #{DYNAMODB_TMP_PATH} -q")
```

Comment:
Warning! Subjective bike-shed ahead!

I think I like something like this better:
```ruby
system("wget #{AMAZON_LOCAL_DYNAMODB_URL} -O #{DYNAMODB_TMP_PATH} -q") or raise "Error downloading #{AMAZON_LOCAL_DYNAMODB_URL}"
```
or
```ruby
x = system("wget #{AMAZON_LOCAL_DYNAMODB_URL} -O #{DYNAMODB_TMP_PATH} -q")
raise "Error downloading #{AMAZON_LOCAL_DYNAMODB_URL}" unless x
```
or ... dunno.
I think the thing I don't like is that "the primary thing" (the side effect `system()`) is "kinda hidden away" in the end of "the line".

Reply:
Feel free to ignore this!

Reply:
I prefer the second option and completely agree with make the `system()` the primary command.